modules/sandboxed-containers-common-terms.adoc

```diff
@@ -0,0 +1,27 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/understanding_sandboxed_containers.adoc
+[id="sandboxed-containers-common-terms_{context}"]
+= {sandboxed-containers-first} common terms
+
+The following terms are used throughout the documentation.
+
+Sandbox:: A sandbox is a controlled environment where programs can run. Within the controlled environment, you can run untested or untrusted programs without risking harm to the host machine or the operating system. A sandbox keeps your workloads isolated from the rest of the system.
++
+In the context of {sandboxed-containers-first}, sandboxing is achieved by running workloads in a different kernel using virtualization, providing enhanced control over the interactions between multiple workloads that run on the same host.
+
+Pod:: A pod is a Kubernetes and {product-title} construct that represents resources where containers can be deployed. Containers run inside of pods, and pods are used to specify resources that can be shared between multiple containers.
```

Comment:
Could this definition compare and contrast pods (general) vs. pods (this document)? 

Also, do we need to call out that it's both k8s + OCP, or can k8s be dropped? Not sure, myself. :)

Reply:
Since OCP is not introducing a `pod` resource on its own and is in fact a rebase of Kubernetes (using it underneath), we initially thought it's okay to use kuberetes. It might be easier to reference an existing definition in OpenShift documentation to prevent though contention to map to a project vs. a product.

Reply:
I understand @zanetworker's rationale, but I think that it does not hurt to write "A pod is a Kubernetes, OpenShift and {product-title} construct...". It does not seem overly verbose.

Or alternatively, "A pod is a {product-title} construct inherited from Kubernetes and OpenShift that represent resources where containers can be deployed." This wording may help clarify the points made in the next sentence that we are explaining how they work in the case of sandboxed containers.